content/webapp/components/ImagePlaceholder/ImagePlaceholder.tsx

```diff
@@ -4,7 +4,7 @@ import { transparentGif, repeatingLs } from '@weco/common/utils/backgrounds';
 import styled from 'styled-components';
 
 const Wrapper = styled.div<{
-  color: ColorSelection;
+  color: ColorSelection | undefined;
```

Comment:
Do we need this undefined? - I think this value comes from the getSeriesColor function which always returns a value

Reply:
it comes from `content/webapp/components/SearchResults/SearchResults.tsx` where it can come back as undefined as it looks like it can be applied to an array of components... Maybe this would be part of the further fix when we apply the right color to the article page?

Reply:
ok yeah, looks like some types need updating elsewhere, but this won't hurt to go in

Reply:
We'll definitely have a little cleanup to do when fixing these issues